src/KineticsStripOutputNode.cpp

```diff
@@ -0,0 +1,102 @@
+#include "KineticsStripOutputNode.h"
+#include <iostream>
+#include <algorithm>
+
+// Reusable state for sending data packets
+static int out_fd = -1;
+// static struct sockaddr_in out_addr;
+
+// This is dynamic because we need to allocate memory based on
+// the number of pixels per strip.
+unsigned char out_packet[512] = {};
+
+std::string ip;
+int startport, width, height;
+struct sockaddr_in out_addr;
+
+KineticsStripOutputNode::KineticsStripOutputNode(Context *context, QSize chainSize)
+: SelfTimedReadBackOutputNode(context, chainSize, 10) {
+
+  connect(this, &SelfTimedReadBackOutputNode::frame, this, &KineticsStripOutputNode::onFrame, Qt::DirectConnection);
+
+  // Try to open a socket
+  if (out_fd > 0) {
+    close(out_fd);
+  }
+  out_fd = socket(AF_INET, SOCK_DGRAM, 0);
```

Comment:
Can you replace these C calls with `QUdpSocket`? http://doc.qt.io/qt-5/qudpsocket.html